.travis.yml

```diff
@@ -1,11 +1,14 @@
 # sudo/dist/group must be required/trusty/edge to access oraclejdk9 (2016-11-14)
```

Comment:
The comment still mentions "edge" although you now deleted it, indicating it might not be quite as required as stated.